assets/component-facets.css

```diff
@@ -291,6 +320,18 @@
   word-break: break-word;
 }
 
+/* Hover, active, and focus states */
+:where(.facets__label:hover, .facets__label.active, .facets__label:has(:focus-visible)) {
+  color: rgba(var(--color-foreground), 1);
+}
+
+/* Focus states for older browsers */
+@supports not selector(:has(a, b)) {
+  .facets__label:focus-within {
+    color: rgba(var(--color-foreground), 1);
+  }
+}
```

Comment:
(question) Couldn't we just put the rule as one of the rules in the `:where` since where is a [forgiving selector parsing](https://developer.mozilla.org/en-US/docs/Web/CSS/:where#forgiving_selector_parsing)